domain/src/main/java/com/distillery/android/domain/FakeToDoRepository.kt

```diff
@@ -122,4 +122,11 @@ class FakeToDoRepository : ToDoRepository {
         }
         todosChannel.close(IllegalArgumentException("You died"))
     }
+
+    override suspend fun deleteToDo(uniqueId: Long) {
+        delay(DELAY_FOR_VALUE_ADDITION)
```

Comment:
The constant name doesn't correspond to its purpose anymore. It's better to either add another constant for the deletion or make the name more generic, like this:
```suggestion
        delay(DELAY_FOR_TODO_OPERATION)
```